packages/components/src/akismet-logo/index.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from 'classnames';
+
+type Props = {
+	className?: string;
+	size?: number;
```

Comment:
Using `size` instead of `height` prop for consistency with `JetpackLogo` component:

https://github.com/Automattic/wp-calypso/blob/f4b610133e816731afbf9e727df935ac17ca238b/client/components/jetpack-logo/index.jsx#L51